org.tenacityaudio.Tenacity.yaml

```diff
@@ -0,0 +1,176 @@
+app-id: org.tenacityaudio.Tenacity
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: tenacity
+rename-icon: tenacity
+rename-desktop-file: tenacity.desktop
+rename-appdata: tenacity.metainfo.xml
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --filesystem=xdg-run/pipewire-0
+  - --system-talk-name=org.freedesktop.RealtimeKit1
+  - --device=all
+  - --filesystem=host
```

Comment:
maybe you can restrict to `home`

Reply:
Should we? I feel like it'll cause a lot of problems, since people may want to load files from an external drive.

Reply:
you can had specific permissions for that.